flow/core/rewards.py

```diff
@@ -441,3 +441,52 @@ def miles_per_gallon(env, veh_ids=None, gain=.001):
     mpg /= 1609.0
 
     return mpg * gain
+
+
+def instantaneous_power(env, veh_ids=None, gain=.001):
```

Comment:
i don’t think we want to re-instantiate the class every time this function is called, so you should find a way to instantiate it at the beginning of the simulation, at the level of VehicleParams